Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'xcpretty'
 
 # A reference to Maze Runner is only needed for running tests locally and if committed it must be
 # portable for CI, e.g. a specific release.  However, leaving it commented out would mean quicker CI.
-gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v6.0.0'
+gem 'bugsnag-maze-runner', git: 'https://github.com/bugsnag/maze-runner', tag: 'v6.3.0'
```

Comment:
Why not use the latest release - [6.5.0](https://github.com/bugsnag/maze-runner/releases/tag/v6.5.0)?

Reply:
No reason, the branch was just sat there for a little while and I didn't update it after the new releases came out.

Reply:
Now updated